packages/mdc-textfield/mdc-text-field.scss

```diff
@@ -655,6 +653,27 @@
   }
 }
 
+.mdc-text-field--outlined:not(.mdc-text-field--upgraded) {
+  height: 56px;
+
+  .mdc-text-field__input {
+    @include mdc-text-field-outlined-corner-radius($mdc-text-field-border-radius);
+
+    height: 100%;
+    padding: 0 0 0 12px;
+    transition: mdc-text-field-transition(border);
```

Comment:
See comment below about transitioning on border / border-color, so I'm wondering if we should forego transition on this. Depending on what your primary color is, transitioning between hover and focus can be rather subtle anyway.